### PR TITLE
Feature/frontend

### PR DESCRIPTION
**Pull request by NathHorrigan**

### CODE REVIEW

File: frontend/app/stories/components.stories.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+
+import { storiesOf } from '@storybook/react'
+import { action } from '@storybook/addon-actions'
+import backgroundColor from 'react-storybook-decorator-background'
+
+import Logo from '../src/components/logo'
```
[the remark this answers is not in the document]
Comment (NathHorrigan) [reply frontend/app/stories/components.stories.js]: Yes. Been meaning to do this

File: frontend/app/storybook-static/iframe.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        }
+    </script>
+    <title>Storybook</title>
+    
+</head>
+
+<body>
+    <div id="root"></div>
+    <div id="error-display"></div>
+<script type="text/javascript" src="static/preview.fabfc854dff67b31b356.bundle.js"></script></body>
```
[the remark this answers is not in the document]
Comment (NathHorrigan) [reply frontend/app/storybook-static/iframe.html]: Generated directory not my code

File: frontend/app/src/components/input-field/index.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import './field.scss'
+
+export default class InputField extends React.Component {
+	state = { focused: false }
```
[the remark this answers is not in the document]
Comment (tm-kn) [reply frontend/app/src/components/input-field/index.js]: You are right, JS will shallow copy this dictionary.

File: frontend/app/storybook-static/iframe.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (tm-kn) [reply frontend/app/storybook-static/iframe.html]: Should we not add it to the repository and add it to .gitignore then?

[the remark this answers is not in the document]
Comment (NathHorrigan) [reply frontend/app/storybook-static/iframe.html]: Deleted

File: frontend/app/yarn-error.log [hunk 4]
```diff
@@ -0,0 +1,9327 @@
+Arguments:
```
[the remark this answers is not in the document]
Comment (NathHorrigan) [reply frontend/app/yarn-error.log]: sure

**Pull request merged**